packages/sdk/src/services/credential-service.ts

```diff
@@ -38,33 +38,54 @@ export class CredentialService extends Service<CredentialsApi> {
    * @param organisationID - signing organisation ID
    * @param key - private key bytes in a form of Uint8Array
    * @param alg - SigningAlg enum value
+   * @param jwtFormat - optional parameter to specify JWT format (default is VC-JWT)
    * @returns Promise<{credential: string; metadata: {style: {"text-color": string, background: string, image: string}}}>
    */
   public async issue(
     auth: IVCToken,
     payload: GenerateCredentialExtendedDto,
     key: Uint8Array,
-    alg: SigningAlg
+    alg: SigningAlg,
+    jwtFormat?: CredentialsControllerGenerateAcceptEnum
   ) {
     if (!auth.organisation_id) {
       throw new MeecoServiceError(
         'credentials.organisation_id parameter is required to issue a credential'
       );
     }
 
```

Comment:
instead of having checks like `jwtFormat && something_else` before all the logic you could write something like this: `jwtFormat = jwtFormat || CredentialsControllerGenerateAcceptEnum.VcJwt` to set the default value if none was provided